pulumi/src/cluster/index.ts

```diff
@@ -100,6 +100,23 @@ export = async (): Promise<Outputs> => {
         cacheFroms: [`${blockbookImage}:${blockbookTag}`, `${blockbookImage}:latest`],
       })
     }
+
+    const volumeReaperImage = `${config.dockerhub.username}/volumereaper`
+    const buildArgs = { BUILDKIT_INLINE_CACHE: '1', BASE_IMAGE: baseImage }
+
+    await buildAndPushImage({
+      image: volumeReaperImage,
+      context: '../../volumeReaper',
+      auth: {
+        password: config.dockerhub.password,
+        username: config.dockerhub.username,
+        server: config.dockerhub.server,
+      },
+      buildArgs: buildArgs,
+      env: { DOCKER_BUILDKIT: '1' },
+      tags: ['test-hash'],
```

Comment:
ideally we folder hash the contents of `volumeReaper` and only build and push if there are changes, using hash as the tag. you can see a reference here: https://github.com/shapeshift/unchained/blob/72f0eccfdf5ecd579567619b92426015adcf0335/pulumi/src/hasher.ts